### ISSUE TITLE
Nixops Broken Because `jobs.*` compatibility layer was removed from nixpkgs

### ISSUE THREAD

**Issue by kevincox** [NixOS/nixops#382]
The PR https://github.com/NixOS/nixpkgs/pull/12198 removed this compatibility layer causing breakage including the following:

```
building all machine configurations...
error: while evaluating the attribute ‘buildCommand’ of the derivation ‘nixops-machines’ at /home/kevincox/p/nixpkgs/pkgs/build-support/trivial-builders.nix:10:14:
while evaluating anonymous function at /nix/store/skc62c32sbwvn2b5mqx5i3cfgw7d6psi-nixops-2015-12-18/share/nix/nixops/eval-machine-info.nix:160:46, called from /home/kevincox/p/nixpkgs/lib/attrsets.nix:165:52:
while evaluating the attribute ‘config.system.build.toplevel’ at /home/kevincox/p/nixpkgs/nixos/lib/eval-config.nix:53:5:
while evaluating the attribute ‘config’ at /home/kevincox/p/nixpkgs/lib/modules.nix:86:25:
while evaluating ‘yieldConfig’ at /home/kevincox/p/nixpkgs/lib/modules.nix:73:29, called from /home/kevincox/p/nixpkgs/lib/modules.nix:72:16:
while evaluating anonymous function at /home/kevincox/p/nixpkgs/lib/modules.nix:79:24, called from /home/kevincox/p/nixpkgs/lib/modules.nix:79:11:
while evaluating anonymous function at /home/kevincox/p/nixpkgs/lib/modules.nix:80:26, called from /home/kevincox/p/nixpkgs/lib/modules.nix:80:13:
The option `jobs' defined in `/nix/store/skc62c32sbwvn2b5mqx5i3cfgw7d6psi-nixops-2015-12-18/share/nix/nixops/ssh-tunnel.nix' does not exist.
error: unable to build all machine configurations
```

If it helps anyone until this issue is resolved, the last working commit of nixpkgs is https://github.com/NixOS/nixpkgs/commit/7c2845128749d1f4a4bf5644695e876193c5932b

**Comment by globin** [comment]
already fixed in #379

**Comment by domenkozar** [comment]
I'll try to make a release.
